File: weather_api/example_code/python/get_sunrise.py
from httpx import Client

with Client() as client:
    response = client.get(
        url="$endpoint_url",
        params={"lat": 52.520008, "lon": 13.404954, "date": "2024-12-24"},
    )

    json = response.json()

    print(f"Sunrise: {json['properties']['sunrise']['time']}")
    print(f"Sunset: {json['properties']['sunset']['time']}")

File: weather_api/example_code/javascript/get_sunrise.js
const response = await fetch(
  "$endpoint_url?lat=52.520008&lon=13.404954&date=2024-12-24"
);
const json = await response.json();

// Get sunrise and sunset times
const sunriseTime = json.properties.sunrise.time;
const sunsetTime = json.properties.sunset.time;

console.log(`Sunrise: ${sunriseTime}`);
console.log(`Sunset: ${sunsetTime}`);
